### PR TITLE
Adds jquery.mb.YTPlayer v2.9.5

### PR DESCRIPTION
**Pull request by Soenneker**
Updated auto-update to reflect latest directory. The css is in a child directory of base though.

I can update the PR if someone can explain how I could include the css in auto-update.

### CODE REVIEW

File: ajax/libs/jquery.mb.YTPlayer/package.json [hunk 1]
```diff
@@ -41,9 +41,10 @@
   "autoupdate": {
     "source": "git",
     "target": "git://github.com/pupunzi/jquery.mb.YTPlayer.git",
-    "basePath": "inc",
+    "basePath": "dist",
     "files": [
-      "jquery.mb.YTPlayer.js"
+      "jquery.mb.YTPlayer.js",
+	  "jquery.mb.YTPlayer.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.mb.YTPlayer/package.json]: wrong indent here

**Comment by Soenneker** [comment]
@PeterDaveHello Auto-update has been fixed, and is ready for merge. Thanks!

**Comment by PeterDaveHello** [comment]
@Soenneker ok, looks better now, would you please squash/fixup the commits to be only one commit? Please also rebase this branch with our latest master branch, its parent is a little bit too old, thanks!

**Comment by PeterDaveHello** [comment]
I added v2.9.3~v2.9.9 here, f16c733, see
https://cdnjs.com/libraries/jquery.mb.YTPlayer/2.9.9

**Pull request closed without merging**